src/app/index.tsx

```diff
@@ -67,9 +104,15 @@ export function App() {
         }}
       />
       {/* <PoolsAndGauges abi={GAUGE_PROXY_ABI} initialDataQuery={INFO_QUERY} /> */}
```

Comment:
remove unnecessary commented out code or please add a description why is commented out.